tracks/swift/exercises/leap/mentoring.md

```diff
@@ -56,3 +56,34 @@ struct Year {
     }
 }
 ```
+
+
```

Comment:
This entire block should probably be under `### Common suggestions`

Reply:
@VigneshkumarG can you move this up?

Reply:
@SleeplessByte Okay

Reply:
@SleeplessByte  Entire block moved under Common suggestions